--- jqplot.core.js.0.9.4	2009-09-08 16:59:57.000000000 +0200
+++ jqplot.core.js	2009-09-08 17:00:25.000000000 +0200
@@ -1283,6 +1283,14 @@
         };
         
         
+        // method: clear
+        // Empties the plot target div.
+        this.clear = function() {
+//            this.target.trigger('jqplotPreRedraw');
+            this.target.empty();
+//            this.target.trigger('jqplotPostRedraw');
+        };
+
         // method: redraw
         // Empties the plot target div and redraws the plot.
         // This enables plot data and properties to be changed
